src/main/java/org/opentripplanner/middleware/tripmonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -431,7 +431,10 @@ private void sendNotifications() {
             "notifications", notifications.stream()
```

Comment:
This entry needs to be updated, it does not match `createTemplateRenderingTestCases` in the test. Try passing `notifications` as is (it is a `Set`), otherwise pass a `List` constructed from it.

Reply:
Let me know if my recent change is okay